octodns_ns1/__init__.py

```diff
@@ -283,6 +283,14 @@ def _try(self, method, *args, **kwargs):
                 )
                 sleep(period)
                 tries -= 1
+            except ResourceException as e:
+                self.log.debug(
+                    "_try: method=%s, args=%s, error=%s",
+                    method.__name__,
+                    str(args),
+                    e.message,
+                )
```

Comment:
Seems reasonable. Switching the debug to an `exception` and removing the `e.message` bit as I believe `self.log.exception` should take care of that and always print w/o having to add `--debug` which makes sense for an error. 

```suggestion
                self.log.exception(
                    "_try: method=%s, args=%s, error=%s",
                    method.__name__,
                    str(args),
                )
```

Reply:
Yeah that makes sense, I can add that change. 

I would like to log the response and body to view information about the request.

```
            except ResourceException as e:
                self.log.exception(
                    "_try: method=%s, args=%s, response=%s, body=%s",
                    method.__name__,
                    str(args),
                    e.response,
                    e.body,
                )
                raise
```

```
2023-10-25T15:53:37  [13073186816] ERROR NS1Client _try: method=retrieve, args=('olinkedin.com',), response=<Response [401]>, body={"message":"Invalid NSONE Key","details":[{"type":"request-id","message":"0a9ed58c-310b-4b70-ae4d-ad7543c12d68"}]}
```